From 1cdc7e119a0137f5028614db3bcebb6063d1f72a Mon Sep 17 00:00:00 2001
From: Matt Fleming <matt.fleming@intel.com>
Date: Thu, 3 Apr 2014 09:51:23 +0100
Subject: [PATCH 1/4] efi_runtime: Set default value for $KVER

Instead of requiring a value to be specified for $KVER, we can set a
default which will do the right thing in most cases, e.g. build the
efi_runtime module against the running kernel.

Cc: Borislav Petkov <bp@alien8.de>
Signed-off-by: Matt Fleming <matt.fleming@intel.com>
---
 efi_runtime/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/efi_runtime/Makefile b/efi_runtime/Makefile
index a9c0ea7..b145502 100644
--- a/efi_runtime/Makefile
+++ b/efi_runtime/Makefile
@@ -1,3 +1,4 @@
+KVER ?= `uname -r`
 obj-m += efi_runtime.o
 all:
 	make -C /lib/modules/$(KVER)/build M=`pwd` modules
-- 
1.7.11.7

